pepys_admin/utils.py

```diff
@@ -202,3 +204,40 @@ def database_at_latest_revision(db_path):
         return True
     else:
         return False
+
+
+class StdoutAndFileWriter:
+    def __init__(self, filename):
+        self.terminal = sys.stdout
+        self.log = open(filename, "a")
+
+    # Pretend not to be a terminal
```

Comment:
@robintw - Codecov says that `isatty` isn't called in the tests.  Does that suggest it's not performing the role you expect it to do?

Reply:
Oh, and good old `Codacy` said this:
![image](https://user-images.githubusercontent.com/1108513/133580840-74271268-3e29-45bf-8568-5b43979ded1b.png)

Reply:
Oh dear - I don't like the fix for the Codacy warning (adding an annotation) - I'm happy to ignore that one.